gensim/models/poincare.py

```diff
@@ -1383,6 +1383,11 @@ def __init__(self, file_path, encoding='utf8', delimiter='\t'):
         ----------
         file_path : str
             Path to file containing a pair of nodes (a relation) per line, separated by `delimiter`.
+            Since the relations are asymmetric, the order of `u` and `v` nodes in each pair matters.
+            To express a "u is v" relation, the lines should take the form `u delimeter v`.
+            e.g: `kangaroo	mammal` is a tab-delimited line expressing a "`kangaroo is a mammal`" relation.
+            `gensim/test/test_data/poincare_hypernyms.tsv
```

Comment:
```suggestion

            For a full input file example, see `gensim/test/test_data/poincare_hypernyms.tsv <https://github.com/RaRe-Technologies/gensim/blob/master/gensim/test/test_data/poincare_hypernyms.tsv>`_.
```

Reply:
Do you know how can I add a line break to the docstring?

> file_path (str) – Path to file containing a pair of nodes (a relation) per line, separated by delimiter. Since the relations are asymmetric, the order of u and v nodes in each pair matters. To express a “u is v” relation, the lines should take the form u delimeter v. e.g: kangaroo mammal is a tab-delimited line expressing a “kangaroo is a mammal” relation. gensim/test/test_data/poincare_hypernyms.tsv.
> 
I don't like how the description is currently shown!